src/typography/typography.css

```
@@ -15,6 +15,15 @@ body {
   text-decoration: underline;
 }
 
+.omrs-title-1 {
+  font-style: normal;
+  font-weight: bold; /* should we be using a variable here to .omrs-bold */
```

I don't feel like the word `bold` warrants the creation of a variable. I just updated the jira issue not to say that because it doesn't make sense. Just use `bold`

```suggestion
  font-weight: bold;
```